source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -96,7 +96,15 @@ Note that a given content host may be classified as having multiple Host_Types a
 Data Types
 """"""""""
 
-R5 reports on scholarly information in many ways. These major groupings, referred to as Data_Types, are listed in the table below along with the Host_Types and reports that they apply to. All Data_Types apply to the Platform Reports since they summarize the usage on the platform. Note that the table lists only Host_Types required to provide one or more reports for compliance, but that report providers may offer additional reports. For example, Host_Type eJournal might also offer IR and IR_A1 and would then use Data_Type Article in these reports.
+To facilitate flexible reporting, R5 introduced Data_Types, representing major groupings of content. The list of Data_Types was extended for R5.1 and are listed in Table 3.p below.
+
+The table lists the Data_Types and related Host_Types which use them in one or more reports for compliance, but Host_Types may choose to offer additional reports. For example, we encourage all Host_Types to offer the Global Item Report using all relevant Data_Types.
+
+Report providers MUST report metrics in line with the following:
+
+* Host_Types required to provide the TR MUST deliver title level information in the PR (e.g. Journal for a journal article or Book for a book). If the DR is also required, usage MUST be reported at the title level within the DR.
+* Host_Types required to provide the TR and which choose to also offer the IR MUST report usage at the item level in IR (e.g. Article for a journal article, Book_Segment for a book).
+* Host_Types that are required to provide the IR, MUST also report at item level in the PR and, if required, the DR.
```

Comment:
I would suggest to add a bullet point that only certain Data_Type/Parent_Data_Type combinations are allowed with a reference to the table below, so that this isn't overlooked.